javascript-light-bulb/main.js

```diff
@@ -0,0 +1,15 @@
+var $lightBulb = document.querySelector('.light-bulb-on');
+var onSwitch = 0;
+
+function bulbClick(event) {
+  onSwitch++;
+  if (onSwitch % 2 === 0) {
```

Comment:
Instead of using a counter variable and just checking if the current count is even or odd, there is a better data type that has only two states that you can use to determine if the light is "on" or not: a boolean.